src/cui.cpp

```diff
@@ -309,6 +354,35 @@ void ui_tick() {
   }
 }
 
+/**
+ * Writes the given lines to the file path passed in from the -o 
+ *  command line option in JSON format.
+ */
+void write_file(Line *lines[], int nproc) {
+  // Get the current timestamp in milliseconds
+  struct timeval tp;
+  gettimeofday(&tp, NULL);
+  long long now = (long long) tp.tv_sec * 1000L + tp.tv_usec / 1000;
+
+  std::ofstream file;
+  file.open(fileoutputpath.c_str());
+
```

Comment:
Do we want to do any assertions that file.open worked properly and didn't have a failure? We did this a ton in school, maybe it isn't necessary but it might be good

Reply:
http://www.cplusplus.com/reference/ios/ios/fail/ or http://www.cplusplus.com/reference/ios/ios/bad/